core/lib/zksync_core/src/consensus/payload.rs

```diff
@@ -0,0 +1,92 @@
+use anyhow::Context as _;
+use zksync_consensus_roles::validator;
+use zksync_protobuf::{required, ProtoFmt};
+use zksync_types::api::en::SyncBlock;
+use zksync_types::{Address, L1BatchNumber, Transaction, H256};
+
+pub(crate) struct Payload {
```

Comment:
`types` crate is imported by virtually everything. I don't think that's the desired level of visibility (here Payload can be even crate-private). ConsensusBlockFields is there only because it is embedded in Block. Otherwise it would be better suited in the `dal` crate.

Reply:
I'm not 100% sure about future use cases, but I could imagine a scenario in which other crates would want to take a look at the (consensus) block payload. Alternatively, we may want to split `zksync_core` into multiple crates in the longer term. But without concrete evidence, these are just hypotheses, so it's fine to leave this definition as-is.